main.js

```diff
@@ -0,0 +1,97 @@
+const MenuSection = document.getElementById('Menu');
+
+const showMenu = () => {
+  MenuSection.style.display = 'block';
+};
+const hideMenu = () => {
+  MenuSection.style.display = 'none';
+};
+
+const speakerContainer = document.querySelector('.home-page');
+
+speakerContainer.innerHTML += `
+<div class="featured-speakers">
+<div class="featured">
+    <img src="img/speaker_01 1.png" alt="" class="speaker-img speaker-img-mobile">
+    <img src="img/speaker_01 1 (7).png" alt="" class="speaker-img speaker-img-desk ">
+    <div class="featured-text">
+            <div class="featured-part1">
+                <div> <p class="third-text1">Yochai Benkler</p></div>
+                <div><p class="third-text2">Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</p></div>
+                <img src="img/title_underline (1).png" alt="">
+            </div>
+            <p class="featured-part2">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+    </div>
+   
+</div>
```

Comment:
- [ ] Your card design appears visually appealing. However, from a semantic HTML perspective, it is recommended to use the `<article>` tag instead of the `<div>` tag for structuring the card content. By employing the appropriate HTML tag, you enhance the semantic meaning of your code and contribute to better accessibility and organization. Please consider making this adjustment to align with best practices. Great job, and keep up the good work! 🌟📝🚀